ts/indicators/bollinger-bands.src.ts

```diff
@@ -0,0 +1,282 @@
+/**
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+import H from '../parts/Globals.js';
+
+/**
+ * Internal types
+ * @private
+ */
+declare global {
+    namespace Highcharts {
+        class BbIndicator extends SmaIndicator
```

Comment:
After internal discussion, and check with speech synthesizers and JavaScript built-in types, this has to be named `BBIndicator` (sub types too).

Reply:
Do not forget to rename the parent to `SMAIndicator` as well.

Reply:
Will be done in a separate PR.